cnv/cnv_release_notes/cnv-release-notes.adoc

```diff
@@ -52,6 +52,12 @@ include::modules/technology-preview.adoc[leveloffset=+2]
 
 == Known issues
 
+* KubeMacPool is disabled in {CNVProductName} {CNVVersion}. This means that the secondary
```

Comment:
should KubeMacPool be in backticks? (I don't know.)

Reply:
Good question - Petr says it's a component, so no markup.